scanner/src/main/kotlin/experimental/ScanController.kt

```diff
@@ -87,10 +89,14 @@ class ScanController(
     private val scanResults = mutableMapOf<ScannerWrapper, MutableMap<KnownProvenance, MutableList<ScanResult>>>()
 
     fun addProvenanceResolutionIssue(id: Identifier, issue: OrtIssue) {
+        logger.warn { issue.message }
```

Comment:
I believe we can't just always use `warn` here but should try to match the issue's severity. So basically do something similar to what we already do in `createAndLogIssue`:

    logger.log(issue.severity.toLog4jLevel()) { issue.message }

Same below.